binaries/workerserver/main.go

```diff
@@ -8,7 +8,8 @@ import (
 
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/scootdev/scoot/common/endpoints"
-	"github.com/scootdev/scoot/runner/execer/fake"
+	"github.com/scootdev/scoot/runner/execer/execers"
+	osexec "github.com/scootdev/scoot/runner/execer/os"
```

Comment:
Did we decide against having everything in ./runner/execer, or the interface in that dir and *all* implementations in ./runner/execer/execers? While the asymmetry of having multiple execers in one dir and prod execer in another does signify intent, it doesn't truly seem necessary. Glad to be rid of 'fake' either way.